plugins/scsi/fu-scsi-device.c

```diff
@@ -6,20 +6,50 @@
 
 #include "config.h"
 
+#ifdef HAVE_BSG_H
```

Comment:
maybe change the plugin_scsi meson.build to auto-enable when these headers are present instead?

Reply:
Hmm, I think we still want the plugin even in read-only mode, e.g. to detect those HEP disks that self destruct.

Reply:
> Hmm, I think we still want the plugin even in read-only mode, e.g. to detect those HEP disks that self destruct.

what is the goal for that info though?

Reply:
We show that info in the "fwupdmgr get-devices" IIRC.

Reply:
If they're not updatable they're hidden by default though. So someone really has to be looking for such a device.